test/stability/testDivide/testDivide.go

```diff
@@ -287,6 +263,45 @@ func executeTest(configFile, target string, testScenario uint, testType uint) er
 	return nil
 }
 
+func setMainTest(inFlow *flow.Flow) (temp *flow.Flow, flow2 *flow.Flow, err error) {
```

Comment:
Why declare named return values and not use it? It is confusing, because you return vars explicitly, dont use temp...
I think it should not be named return values - just local flow1, flow2 vars